agent/browser/ie/wptbho/wptbho.cc

```diff
@@ -85,6 +86,7 @@ STDMETHODIMP_(void) WptBHO::OnDocumentComplete(IDispatch *pDisp,
   CComPtr<IUnknown> unknown_browser = _web_browser;
   CComPtr<IUnknown> unknown_frame = pDisp;
   if (unknown_browser && unknown_frame && unknown_browser == unknown_frame) {
+    _wpt.OnDocumentComplete();
     _wpt.OnLoad();

```

Comment:
Good call. There isn't an onload hook in BHO anyway. I'll create 2 hooks: `OnDocumentReady` and `OnDocumentComplete`.